src/main/play-25/uk/gov/hmrc/hmrcfrontend/config/ContactFrontendConfig.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hmrcfrontend.config
+
+import javax.inject.Inject
+import play.api.Configuration
+import play.api.mvc.RequestHeader
+
+class ContactFrontendConfig @Inject()(config: Configuration) {
+  private val platformHost: Option[String] =
+    config.getString("platform.frontend.host")
+
+  val baseUrla: Option[String] =
+    platformHost.orElse(config.getString("contact-frontend.host"))
+  val serviceCodea: Option[String] =
+    config.getString("contact-frontend.serviceCode")
+
+  def referrerUrl(implicit request: RequestHeader): Option[String] = {
+    val query = if (request.rawQueryString != "") { s"?${request.rawQueryString}" } else { "" }
+    Some(s"${platformHost
+      .getOrElse("")}${request.path}$query")
+  }
+  def baseUrl(): Option[String] =
```

Comment:
This definition can be removed as you already have a val above (although it needs renaming).